.github/PULL_REQUEST_TEMPLATE/feature.md

```diff
@@ -0,0 +1,41 @@
+## Summary
+
+Briefly describe the feature being introduced.
+
+## Rationale
+
+Explain the reasoning behind this feature and its benefits to the project.
+
+## Design Documentation
+
+Link to any design documents or diagrams relevant to this feature.
+
+## Changes
+
+List the major changes made in this pull request.
+
+## Impact
+
+Discuss any potential impacts this feature may have on existing functionalities.
+
+## Testing
+
+Describe how the feature has been tested, including both automated and manual testing strategies.
+
+## Screenshots/Video
+
+Include screenshots or video demonstrating the new feature, if applicable.
+
+## Checklist
+
+- [ ] Code follows the project's coding standards
```

Comment:
Are these formalized somewhere? There are some aspects in the README - Changelog files, linting